sample/AspNetFullFrameworkSampleApp/App_Start/ApmLoggerToNLog.cs

```diff
@@ -8,17 +8,33 @@ namespace AspNetFullFrameworkSampleApp
 {
 	internal class ApmLoggerToNLog : IApmLogger
 	{
-		public bool IsEnabled(ApmLogLevel level) => true;
+		private readonly Lazy<Logger> _logger;
+
+		public bool IsEnabled(ApmLogLevel level) => _logger.Value.IsEnabled(ConvertLevel(level));
+
+		public ApmLoggerToNLog()
+			:this(string.Empty)
+		{
+
+		}
+
+		public ApmLoggerToNLog(string loggerName)
+		{
+			_logger = new Lazy<Logger>(() => LogManager.GetLogger(loggerName ?? string.Empty), true);
+		}
```

Comment:
nit: [we use](https://github.com/elastic/apm-agent-dotnet/blob/master/.editorconfig#L70) expression bodied .ctor for 1 liners

```suggestion
		public ApmLoggerToNLog(string loggerName) => _logger = new Lazy<Logger>(() => LogManager.GetLogger(loggerName ?? string.Empty), true);
```

Reply:
Looks more convoluted , but your wish, my command.